README.adoc

```diff
@@ -112,34 +112,41 @@ include::finish/src/main/java/io/openliberty/guides/consumingrest/model/Artist.j
 include::finish/src/main/java/io/openliberty/guides/consumingrest/model/Album.java[tags=!comment]
 ----
 
-=== Using Jackson
-
-In the case of using Jackson for JSON binding, a few restrictions are applicable to these objects,
-they:
+=== Using JSON-B
 
-* Cannot have a constructor unless it is empty.
-* Must contain the same number of instance members as there are keys in the target JSON.
+JSON-B is a new feature introduced with Java EE 8 and strengthens Java's support for JSON.
```

Comment:
- Let's omit the word "new." You already say that it's introduced with Java EE 8, and that statement should suffice. "New" can be such a relative term and might feel outdated if this guide stays around a while.
- I also suggest, "Java support" instead of "Java's support."